src/server/game/Entities/Player/Player.cpp

```diff
@@ -24166,6 +24166,30 @@ inline void BeforeVisibilityDestroy<Creature>(Creature* t, Player* p)
         t->ToPet()->Remove(PET_SAVE_NOT_IN_SLOT, true);
 }
 
+void Player::UpdateVisibilityOf(std::list<WorldObject*> targets)
```

Comment:
Do not use an owning datastructure here https://en.cppreference.com/w/cpp/container/span (if this is available to TC)

Reply:
Tc is using cpp17, not 20